draft-ietf-quic-transport.md

```diff
@@ -357,7 +357,6 @@ Within each type, streams are created with numerically increasing stream IDs.  A
 stream ID that is used out of order results in all streams of that type with
 lower-numbered stream IDs also being opened.
 
-The first bidirectional stream opened by the client has a stream ID of 0.
```

Comment:
Removed on what basis?  I know that this *seems* intuitive, and we do say 0..2^62-1 but an explicit statement that identifies the first value isn't harmful, is it?

Or are you saying that we have too much baggage and we shouldn't let that show so much?

Reply:
It reads to me like baggage; it's only notable because it didn't used to be true.  On a fresh read without the mental baggage, it just feels strange to call *that one type* out.  The alternative to me would be identifying the first one for each type, but since that's simply the low-order bits with the higher bits zero, it felt superfluous.